### PR TITLE
Start cleaning up a little to simplify some further refactoring

### PR DESCRIPTION
**Pull request by jflatow**
Eventually setting up for idempotent deploy

### CODE REVIEW

File: plugins/import/import.ts [hunk 1]
```diff
@@ -14,7 +13,7 @@ export async function loadContract(source: string, network: string, address: str
   }
   switch (source) {
     case 'etherscan':
-      return await loadEtherscanContractMemoized(network, address);
+      return await loadEtherscanContract(network, address);
```
Comment (scott-silver) [inline plugins/import/import.ts:16]: i think we're going to miss memoizing this, no?

Comment (jflatow) [reply plugins/import/import.ts:16]: I don't ever see it being a hit, and probably because we already have multiple levels of caching

Comment (scott-silver) [reply plugins/import/import.ts:16]: 👌

**Review by scott-silver — APPROVED** [review]

**Review by antonina-cherednichenko — APPROVED** [review]

**Comment by jflatow** [comment]
Covered by #517 #520 and #521

**Pull request closed without merging**